runtime/src/iree/runtime/CMakeLists.txt

```diff
@@ -8,7 +8,9 @@
 # To disable autogeneration for this file entirely, delete this header.        #
 ################################################################################
 
-iree_add_all_subdirs()
+if(IREE_BUILD_SAMPLES)
+  add_subdirectory(demo)
+endif()
```

Comment:
> I want to build IREE runtime without any demo, sample, test. I know this commit cannot pass `bazel_to_cmake` check. However, I don't know how to make both of them happy.

Looks like the simplest thing to do would be to edit the `demo/CMakeLists.txt` to have an early return at the start, since _that_ file is not autogenerated: https://github.com/iree-org/iree/blob/main/runtime/src/iree/runtime/demo/CMakeLists.txt

If that file were autogenerated, you could use `BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_ABOVE_THIS_LINE` as is done here: https://github.com/iree-org/iree/blob/2c0a24c0b21a8424a5e7304e294efff8c7e07fed/tools/testing/e2e/CMakeLists.txt#L1-L6